### DISCUSSION TITLE
Support SSA subtitles including advanced styles

### DISCUSSION THREAD

**Discussion by xkww3n** [moneytoo/Player#421] [Ideas]
Now Just Player can only handle the position and content of SSA subtitles. Could you add the support of the advanced style such as font and color? If not, please at least allow users to change the background of the subtitle area.
Btw Just Player is the best video player (yes, not just among the FOSS applications) I've ever seen. Thanks for such a good work.

**Comment by moneytoo** [comment]
See https://github.com/moneytoo/Player#how-do-i-change-subtitle-font-size-or-color & #309

**Reply by xkww3n** [reply]
Ok, thanks for replying.
